cpp/include/cuml/cuml.hpp

```diff
@@ -95,6 +96,13 @@ class cumlHandle {
   * @returns num of streams in the handle.
   */
   int getNumInternalStreams();
+
+  /**
+   * @brief API to get the internal streams as a vector.
+   * @return vector of internal streams in the handle
+   */
+  std::vector<cudaStream_t> getInternalStreams() const;
+
```

Comment:
@cjnolet I think exposing this only from `cumlHandle_impl` makes sense. We wouldn't want users of libcuml++.so to be aware of this concept.

Reply:
This makes sense wrt PV #1396 . Marking as resolved.